README.md

```diff
@@ -179,6 +201,7 @@ styled-system can be configured with styled-components'
 
 ```jsx
 import { ThemeProvider } from 'styled-components'
+// or import { ThemeProvider } from 'glamorous'
```

Comment:
Just wanted to make it clear to people who are coming from glamorous. Happy to remove the styled-components import in favor of glamorous 😉 jkjk, I can remove this if you want.